core/libp2p/protocol/kademlia/pb/kad.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+// https://github.com/libp2p/specs/pull/108
+
+package kad.pb;
+
+// Record represents a dht record that contains a value
+// for a key value pair
+message Record {
+    // The key that references this record
+    bytes key = 1;
+
+    // The actual value this record is storing
+    bytes value = 2;
+
+    // Note: These fields were removed from the Record message
+    // hash of the authors public key
+    //optional string author = 3;
+    // A PKI signature for the key+value+author
+    //optional bytes signature = 4;
```

Comment:
Maybe remove them entirely, then?

Reply:
I just copy-pasted protobuf from spec.

Reply:
Also this comment gives a clue why `timeReceived = 5` and not `=3`